src/Splitter/TestsSplitter.php

```diff
@@ -144,20 +150,23 @@ protected function resolveDependenciesToFullNames(array $testsListWithDependenci
                 $dependency = str_replace('::', ':', $dependency);
                 // className:testName, that means we need to find proper test.
                 [$targetTestFileName, $targetTestMethodName] = explode(':', $dependency);
-
+                if (false === strrpos($targetTestFileName, '.php')) {
+                    $targetTestFileName .= '.php';
```

Comment:
@DavertMik Is it possible that we can received here other type of strings than paths with extensions?

Reply:
Not sure. I hope not :smile: